go/README.md

```diff
@@ -1 +1,177 @@
-# Go
\ No newline at end of file
+# Go Code Conventions
+
+## Tooling
+
+- Visual Studio Code
+- delve
+- golangci-lint
+- staticcheck
+- modd
+
+## Modules
+
+go mod init bitbucket.org/aanzeeonline/[yourprojectname]
+
+Working with a local fork of a module, put an extra line in the go.mod:
+// replace bitbucket.org/aanzeeonline/libgoaz => ../LibGOAZ
+(and remove the comments)
+
+Before pushing the final update to git, make sure that the fork isn't in use anymore
+
+## project structure for an API / Web application
+
+main
+- start (explain why)
+- web
+- helpers
+- datamodels
+- business (separation of concerns)
+  - domains (if you have more than one)
+    - handlers
+    - queries (if you work with a database)
+    - repositories
+    - services
+    - validators
+
+## Makefile
+
+Every project must have a makefile.
+(You can look at other projects for an universal working version)
```

Comment:
Should we standardize the make commands? example would be the `npm run start` and `npm run build`

Reply:
Describe make file "standards"